app/models/jellyfish_demo/service/storage.rb

```diff
@@ -0,0 +1,55 @@
+module JellyfishDemo
+  module Service
+    class Storage < ::Service
+      def actions
+        actions = super.merge :terminate
+        actions
+      end
+
+      def provision
+        random_id = 1 + rand(99_999)
```

Comment:
`rand 2..99_999` and the next line `rand 1..254`

Ruby has `Ranges`, these can be defined as two values with either `..` or `...` between them. http://ruby-doc.org/core-2.3.0/Range.html